### ISSUE TITLE
[Feature Request] Add plot and trailer link

### ISSUE THREAD

**Issue by thundergreen** [custom-cards/upcoming-media-card#12]
Hello thanks a lot for this work I just wanted to know if it is possible to add a link for trailer and plot of the movie ?

**Comment by maykar** [comment]
As discussed on the forums, this is on my todo list. Would possibly be adding these in a pop up window. Since the built in HA more info window wouldn't be item specific, I'll have to get creative.

**Comment by iantrich** [comment]
If you did end up doing this within HA, I would recommend using `paper-dialog`. Here's a recent example for editing LL configs from within the UI. https://github.com/home-assistant/home-assistant-polymer/pull/1885/files

**Comment by CMaxim77** [comment]
Great Documentation, on this! Very detailed and clear, the options you put in the instructions are crazy good! 

Can google calendar events be added to this, I have sensors for upcoming television shows and this would be awesome to list those shows!

Thanks!

**Comment by iantrich** [comment]
@maykar what about just providing an actual link for now to an external site that would open in a new tab? Would be dictated by the component on if that link is populated and the source. e.g. for the trakt component, I would link to the trakt page for the episode and maybe even a separate link for the show itself?

**Comment by mkanet** [comment]
@thundergreen thank you so much for your suggestions. I know its 5 YEARS later; but still better than never.  I just added support to display tooltips.  I also updated Sonarr Upcoming, Radarr Upcoming, and Plex Recently added integrations to provide plots/summaries to Upoming Media Card; just enable the tooltip feature in Upcoming Media Card.  

I added a trailer attribute to Radarr Upcoming Media.  However I wasn't able to think of a good spot to add a clickable hyperlink for each item.
